src/asciidoctor-kroki.js

```diff
@@ -1,3 +1,4 @@
+// @ts-check
```

Comment:
What does this comment do exactly? Since we are not using TypeScript we should probably use `@ts-nocheck` no?

Reply:
This comment enables type checking in JS files by the TypeScript compiler. JSDoc types are used for (optional) type information (https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html). With this comment, the TypeScript compiler is enabled in VS Code and an error would be shown for example, if I forgot an argument when calling `preprocessVegaLite` or if I forgot to return a result from this function (which actually happened to me yesterday). This comment should not disturb in other editors. One could also generally activate (and configure) the TypeScript check for all JS files with the help of a `tsconfig.json` file and explicitly deactivate it if necessary.

Reply:
Ok I was asking because you said "...which helps me to avoid silly errors.". So I thought that you  would have needed to disable TypeScript compiler.
But I'm fine with this comment.